tests/src/system/rest/GoCLINginxTests.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package system.rest
+
+import org.junit.runner.RunWith
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+import org.scalatest.junit.JUnitRunner
+
+import com.jayway.restassured.RestAssured
+
+/**
+ * Basic tests of the download link for Go CLI binaries
+ */
+@RunWith(classOf[JUnitRunner])
+class GoCLINginxTests extends FlatSpec with Matchers with RestUtil {
+
+    val DownloadLinkGoCli = "/cli/go/download"
+    val MacArchitecture = List("386", "amd64", "arm", "arm64")
+    val LinuxArchitecture = List("386", "amd64", "arm", "arm64",
+                                 "mips64", "mips64le", "ppc64", "ppc64le")
+    val WindowsArchitecture = List("386", "amd64")
+
+    //// Check the download page for all the Go Cli binaries
+    it should "respond to " + DownloadLinkGoCli in {
+        val response = RestAssured.given().config(sslconfig).
+            get(getServiceURL() + DownloadLinkGoCli)
+
+        response.statusCode() should be(200)
```

Comment:
tip: by convention, methods that are not side-affecting (as in stausCode) should omit their `()`.